srcpkgs/haproxy/update

```diff
@@ -0,0 +1,2 @@
+# 2.6 is the current LTS
+ignore="2\.[!6].*"
```

Comment:
if every even minor version is an LTS release, we could modify this pattern to match all even minor releases similar to how xbps-src does it for gnome projects by default

Reply:
I'd rather bump that by hand, so that there's some time to work out the kinks in any given release. I'm only bumping the LTS version to `2.6.x` because we're deploying it en masse at $work.